# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/tests/mochitest/general/resource_timing_main_test.html
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/dom/tests/mochitest/general/resource_timing_main_test.html b/dom/tests/mochitest/general/resource_timing_main_test.html
--- a/dom/tests/mochitest/general/resource_timing_main_test.html
+++ b/dom/tests/mochitest/general/resource_timing_main_test.html
@@ -40,16 +40,17 @@
   entries for this iframe's performance object.
   The parent's (this window) "ok_wrapper()" method will be called once the tests
   are completed.
 -->
 
 <!DOCTYPE html>
 <html>
 <head>
+  <meta charset="utf-8">
 	<link rel="stylesheet" href="/tests/SimpleTest/test.css?resource-timing-main-test"/>
 	<script type="application/javascript">
 
 var mainWindowTestsDone = false;
 var iframeTestsDone = false;
 
 function ok(cond, message) {
   window.opener.ok(cond, message)